src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/create_edit_field/create_edit_field.js

```diff
@@ -17,12 +17,52 @@
  * under the License.
  */
 
-import 'ui/field_editor';
 import { IndexPatternsFieldProvider } from 'ui/index_patterns/_field';
+import { RegistryFieldFormatEditorsProvider } from 'ui/registry/field_format_editors';
 import { KbnUrlProvider } from 'ui/url';
 import uiRoutes from 'ui/routes';
 import { toastNotifications } from 'ui/notify';
-import template from './scripted_field_editor.html';
+import template from './create_edit_field.html';
+
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { FieldEditor } from 'ui/field_editor';
+
+const REACT_FIELD_EDITOR_ID = 'reactFieldEditor';
+const renderFieldEditor = ($scope, indexPattern, field, {
+  Field,
+  getConfig,
+  $http,
+  fieldFormatEditors,
+  redirectAway,
+}) => {
+  $scope.$$postDigest(() => {
+    const node = document.getElementById(REACT_FIELD_EDITOR_ID);
+    if (!node) {
+      return;
+    }
+
+    render(
+      <FieldEditor
+        indexPattern={indexPattern}
+        field={field}
+        helpers={{
```

Comment:
I use `helper` for things that are tied up with Angular dependencies - mostly providers that are wrapped with `Private`. This will hopefully make it easier in the future to identify dependencies that can be `import`ed directly once unwrapped from Angular.